lang/en.js

```diff
@@ -234,4 +234,20 @@ export default {
   "is an invalid code": "is an invalid code",
   "promo code applied": "promo code applied",
   Qty: "Qty:",
+  "Checking the status of your order is fast and simple!":
+    "Checking the status of your order is fast and simple!",
+  "Simply enter your order number and billing email to track your order":
```

Comment:
Please keep _key_ on camel case as much as possible instead whole text.
eg. enterOrderNumberAndBillingEmail

Reply:
Keys should also be short, the key used here is the full length of the message

Reply:
Resolved